lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -82,6 +82,7 @@ def __init__(
 
         self.initial_azim = azim
         self.initial_elev = elev
+        self.proj_type = proj_type
```

Comment:
Why this? it gets overwritten by `set_proj_type` in the next line. If you just want to define `self.proj_type` in `__init__`, I'd rather set it to `None`.